operator/src/main/java/com/github/streamshub/console/api/v1alpha1/status/Condition.java

```diff
@@ -61,4 +67,70 @@ public String getMessage() {
     public void setMessage(String message) {
         this.message = message;
     }
+
+    public boolean isActive() {
+        return active;
+    }
+
+    public void setActive(boolean active) {
+        this.active = active;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(message, reason, status, type);
+    }
+
+    /**
+     * For the purposes of equality, we do not consider the lastTransitionTime.
```

Comment:
Is active intentionally omitted from the equality too?

Reply:
Yes, `active` is only meant as a marker for Conditions that still apply at the end of the reconcile process. This helps maintain an accurate `lastTransitionTime` for a given condition. I'll update the comment that it's excluded as well.